cst438-week2/.gitignore

```diff
@@ -6,9 +6,7 @@ target/
 
 ### STS ###
 .apt_generated
-.classpath
 .factorypath
-.project
```

Comment:
This was necessary for Eclipse.

Reply:
Good. Now I can safely import your repo to Eclipse